server/odc-core/src/main/resources/i18n/BusinessMessages.properties

```diff
@@ -193,6 +193,7 @@ com.oceanbase.odc.AuditEventAction.CREATE_SHADOWTABLE_SYNC_TASK=Create shadowtab
 com.oceanbase.odc.AuditEventAction.CREATE_PARTITION_PLAN_TASK=Create partition plan task
 com.oceanbase.odc.AuditEventAction.CREATE_ALTER_SCHEDULE_TASK=Create alter schedule task
 com.oceanbase.odc.AuditEventAction.CREATE_ONLINE_SCHEMA_CHANGE_TASK=Create online schema change task
+com.oceanbase.odc.AuditEventAction.CREATE_PERMISSION_APPLY_PROJECT_TASK=Create permission apply - project task
```

Comment:
the description is kind of confusing by 'permission apply - project task'

Reply:
Done, all identifier like `permission apply - project` will be replaced to `apply project permission`